pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -138,4 +138,38 @@ typedef struct {
     char buf[HTTP2_FRAME_HEADER_SIZE];
 } frame_header_remainder_t;
 
+// http2_telemetry_t is used to hold the HTTP/2 kernel telemetry.
+// request_seen                 Count of HTTP/2 requests seen
+// response_seen                Count of HTTP/2 responses seen
+// end_of_stream_eos            Count of END_OF_STREAM flags seen
+// end_of_stream_rst            Count of RST flags seen
```

Comment:
wondering if we need both, or should we just remove the `rst` field, and increase `eos` everytime we call handle_end_of_stream

Reply:
I believe it could be beneficial to have this metric at the beginning. It would provide us with better insight into whether we are handling these issues correctly. If we encounter issues with other errors in the RST process, this metric would assist us in understanding the challenges we are facing."